src/test/regress/expected/multi_table_ddl.out

```diff
@@ -62,11 +60,20 @@ SELECT * FROM pg_dist_shard_placement;
 ---------+------------+-------------+----------+----------
 (0 rows)
 
--- check that the extension now can be dropped (and recreated). We reconnect
--- before creating the extension to expire extension specific variables which
--- are cached for performance.
+-- check that the extension now can be dropped (and recreated)
 DROP EXTENSION citus;
-WARNING:  could not clean the metadata cache on DROP EXTENSION command
-HINT:  Reconnect to the server again.
-\c
 CREATE EXTENSION citus;
+-- verify that a table can be created after the extension has been dropped and recreated
+CREATE TABLE testtableddl(somecol int, distributecol text NOT NULL);
+SELECT master_create_distributed_table('testtableddl', 'distributecol', 'append');
+ master_create_distributed_table 
+---------------------------------
+ 
+(1 row)
+
+SELECT * FROM testtableddl;
```

Comment:
What if you create a new test for create/drop extension at the end? This seems somewhat unrelated to the other tests.